doc/src/odin/dev_guide/contributing.md

```diff
@@ -18,12 +18,12 @@ Add a description of the changes made and reference the "issue" that it corrects
 
 ### Creating a "Work in progress" (WIP) PR
 
-**Important** Before creating a WIP PR, if it is a bug you have happened upon and intend to fix make sure you create an issue beforhand.
+**Important** Before creating a WIP PR, if it is a bug you have happened upon and intend to fix make sure you create an issue beforehand.
 
 A "work in progress" PR is a pull request that isn't complete or ready to be merged.
 It is intended to demonstrate that an Issue is being addressed and indicates to other developpers that they don't need to fix it.
 Creating a WIP PR is similar to a regular PR with a few adjustements.
-First, clone the vtr-verilog-rooting repository and branch from the master.
+First, clone the [vtr-verilog-rooting repository](https://github.com/verilog-to-routing/vtr-verilog-to-routing) and branch from the master.
```

Comment:
Good catches Joannah!

Do we need to specify this because it can be done through ssh as well; so either we give the path (git@github.com:verilog-to-routing/vtr-verilog-to-routing) for ssh cloning too or we don't provide any, at all. Also, same changes need to be replicated in ## Creating a Pull Request (PR) section. Hope it makes sense!

Typo: vtr-verilog-to-routing repository